test/deprecated/random/fillrandom-bounded.chpl

```diff
@@ -0,0 +1,215 @@
+use Random;
```

Comment:
This one also seems like something that you should adjust to work without using the deprecated interfaces. I think you just need to set the eltType to `uint` since that is what it is asking for.

I think this test is doing something useful even though it's currently written to use the deprecated `getNext(type)`. It can be adjusted to stop doing that.

Reply:
There is a non-deprecated version of this test under `test/library/standard/Random/ferguson`.